src/views/v2/rsvp/ari/sidebar/guest-list/guest-template.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * This template renders the RSVP ARI sidebar guest list item JS template.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/sidebar/guest-list/item-template.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+?>
+<script
+	class="tribe-tickets__rsvp-ar-guest-list-item--template"
```

Comment:
a technicality, but the class should be something like `tribe-tickets__rsvp-ar-guest-list-item-template`, since the <script> tag isn't a modification of the `tribe-tickets__rsvp-ar-guest-list-item` element.